client-src/elements/chromedash-gantt.ts

```diff
@@ -1,27 +1,21 @@
-import {LitElement, css, html, nothing} from 'lit';
+import {LitElement, TemplateResult, css, html, nothing} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {Feature, StageDict} from '../js-src/cs-client.js';
 import {
   STAGE_TYPES_DEV_TRIAL,
   STAGE_TYPES_ORIGIN_TRIAL,
   STAGE_TYPES_SHIPPING,
 } from './form-field-enums.js';
 
+@customElement('chromedash-gantt')
 class ChromedashGantt extends LitElement {
-  static get properties() {
-    return {
-      feature: {type: Object},
-      stableMilestone: {type: Number},
-    };
-  }
-
-  constructor() {
-    super();
-    this.feature = {};
-    this.stableMilestone = null;
-    window.csClient
-      .getChannels()
-      .then(channels => (this.stableMilestone = channels['stable'].version));
```

Comment:
Keep this in the constructor, or in a new [`connectedCallback()`](https://lit.dev/docs/components/lifecycle/#connectedcallback). The component needs to be able to handle it being null, since the API call won't return instantly.